app/lib/BaseEditorController.php

```diff
@@ -700,6 +700,19 @@ public function Summary($pa_options=null) {
 		$vb_ajax_load_displays = $t_subject->getAppConfig()->get('enable_ajax_summary_displays');
 		session_write_close();
 		$this->SummaryData($pa_options, $vb_ajax_load_displays);
+
+		$this->view->setVar($t_subject->tableName().'_summary_last_settings', Session::getVar($t_subject->tableName().'_summary_last_settings'));
+
+		$this->opo_app_plugin_manager->hookSummarizeItem(
```

Comment:
Ah this would have been useful ;) for us. Could potentially have done this using a plugin instead. But nevermind for now.